src/Server/WebServer.fs

```diff
@@ -11,7 +11,9 @@ open Suave.RequestErrors
 open Microsoft.Azure.WebJobs
 open ServerCode.Storage.AzureTable
 
-type DatabaseType = FileSystem | Azure of ConnectionString:AzureConnection
+type DatabaseType = 
+    | FileSystem 
+    | Azure of ConnectionString : AzureConnection
```

Comment:
Why CamelCase for the field name?

Reply:
what do you suggest?

Reply:
`| Azure of connectionString : AzureConnection`